### PR TITLE
Add "Usage templates" page

### PR DESCRIPTION
**Pull request by aguschin**
https://mlem-ai-configuration-p-k9xn6s.herokuapp.com/doc/config

WIP draft for https://github.com/iterative/mlem.ai/issues/190

Wonder, how to fit <table data | cv | nlp> selector here.

---

@yathomasi, thanks for https://github.com/iterative/mlem.ai/pull/238#issuecomment-1335237300, that worked) Don't know what was the reason.

### CODE REVIEW

File: content/docs/sidebar.json [hunk 1]
```diff
@@ -10,6 +10,11 @@
     "source": "install.md",
     "label": "Installation"
   },
+  {
+    "slug": "config",
+    "label": "Usage templates",
```
Comment (jorgeorpinel) [inline content/docs/sidebar.json:15]: Should be in User Guide I think. Let's use a more consistent slug and name too?

**Comment by aguschin** [comment]
Ah, didn't commit the page itself 😂

**Comment by aguschin** [comment]
The summary of a discussion with @omesser and @mike0sv:
1. This page tries to achieve two things: show use cases and provide code templates. In case of MLEM, this can be hard to achieve.
2. The tabs doesn't look balanced: some have a lot of context, some do not.
3. We can put it in different pages and result of adaptation to those locations may be different. In `mlem.ai` main page this should be catchy, while in docs it should be more detailed.
4. Considering we have a lot of pages in docs already and we need to get more attention to MLEM, more useful to put it in mlem.ai main page and have a toggle like in CML https://cml.dev, or have a diagram like in https://lets-unify.ai

Considering this, I'm closing the PR in favor of https://github.com/iterative/mlem.ai/issues/248

**Comment by jorgeorpinel** [comment]
> I'm closing the PR in favor of #248

Hmm, should we merge with #190?

**Comment by aguschin** [comment]
Sorry, didn't get you. Do you mean to close #190? Or what?

**Comment by jorgeorpinel** [comment]
Yes, merge the 2 issues. Is this essentially a duplicate of #190? Thanks

**Comment by jorgeorpinel** [comment]
p.s. Well, now this is closed but some of the info/comments above could be added to #190 if still needed. Up to you, thanks again

**Pull request closed without merging**